backend/app/tests/utils/utils.py

```
@@ -1,6 +1,7 @@
 import random
 import string
 from uuid import UUID
+from typing import List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)